test_data/cloud-init/ubuntu/network-config

```diff
@@ -9,4 +9,4 @@ network:
            address: 192.168.2.2/24
            gateway: 192.168.2.1
            dns_nameservers:
-              - 192.168.2.1
\ No newline at end of file
+              - 192.168.2.1
```

Comment:
Why's this edit in here?

Reply:
A network interface was added in this file for vhost-user-net device previously, with expectation to get IP and mac assigned for it through cloud-init, but seems not work, so I have recovered this file, and some commits will be spuashed finally, this file will be as unchanged.